CloudCollector/src/main/java/com/capitalone/dashboard/model/AWSConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.capitalone.dashboard.model;
+
+public class AWSConfig extends CollectorItem {
```

Comment:
This would be changed in the AWSCloudCollectorTask.java where the ec2Client and the cloud watch client are set up. That file is in a pull request by @nagadeepakc. 

Currently in that file, we used the function, BasicAWSCredentials, which allows you to pass in the access key and secret key directly. Using the DefaultAWSCredentialsProviderChain, we can use the standard locations like the environment variables or the credentials file.